Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM busybox:1.35
+
+RUN mkdir /ezua-tutorials
+
+COPY /Data-Engineering/PrestoDB /ezua-tutorials/Data-Engineering/PrestoDB
+
+COPY /Data-Science /ezua-tutorials/Data-Science
+
+COPY /E2E-Demos /ezua-tutorials/E2E-Demos
+
+COPY /Spark-GPU /ezua-tutorials/Spark-GPU
```

Comment:
Spark-GPU will be moved Data-Analytics dir in upcoming PR.. please remove this line

Reply:
Also, don't we beed to copy Spark examples to shared vol ? I see Airflow is not copied, bcoz it will be integrated directly with Airflow deployment!

Reply:
@prasadadireddi right, Airflow is not copied because it reads (syncs) DAGs from this repo in runtime. Spark part just used by Airflow - docker image of Spark app. We only push new image of app using https://github.com/HPEEzmeral/ezua-tutorials/blob/main/Data-Analytics/Spark/devops/install.sh.
But I'm not sure about SparkGPU example -- maybe we need to upload it to shared also.
@prasadadireddi what do you think?

Reply:
I think, it is good to push them to shared, so that users can consume the yaml files as per heir need!

Reply:
Thanks @prasadadireddi . Done